AdminClient/src/app/dashboard/dashboard.component.scss

```diff
@@ -0,0 +1,4 @@
+mat-grid-tile {
```

Comment:
Should this be an element selector or class selector?

Reply:
Made this for element selectors, just for the dashboard tiles - didn't think I'd use other tiles for dashboard. Can modularise it more if needed, let me know what you think!